pandas_datareader/yahoo/daily.py

```diff
@@ -149,7 +149,7 @@ def _read_one_data(self, url, params):
         try:
             j = json.loads(re.search(ptrn, resp.text, re.DOTALL).group(1))
```

Comment:
if a patter is not found it crashes with different exception:
`AttributeError: 'NoneType' object has no attribute 'group'`